wallet/build.gradle

```diff
@@ -142,6 +146,10 @@ android {
     }
 
     buildTypes {
+        all {
+            buildConfigField("String", "IMGUR_CLIENT_ID", props.getProperty("IMGUR_CLIENT_ID"))
+            buildConfigField("String", "IMGUR_CLIENT_SECRET", props.getProperty("IMGUR_CLIENT_SECRET"))
```

Comment:
This will allow Travis CI to pass, though builds without a local.properities will not have working Imgur functionality.
```
           buildConfigField("String", "IMGUR_CLIENT_ID", props.getProperty("IMGUR_CLIENT_ID", "000000000000000"))
            buildConfigField("String", "IMGUR_CLIENT_SECRET", props.getProperty("IMGUR_CLIENT_SECRET", "00000000000000000000000000000000000000"))
```

Reply:
Actually, this doesn't build, at least not for me.

Reply:
Fixed on https://github.com/dashevo/dash-wallet/pull/547/commits/8956f5ad0e8f43f6765c4a2adefcb9c1cdbde31a